R/config.R

```diff
@@ -371,7 +375,7 @@ default_config_dir <- function(os = NULL) {
     unix = "~/.config/snowflake"
   )
 
-  os_paths[[os]]
+  path.expand(os_paths[[os]])
```

Comment:
The underlying reason may be similar to https://github.com/rstudio/rsconnect/issues/1152.